### PR TITLE
Coverage improvements

### PR DESCRIPTION
**Pull request by bjornhellander**
I checked code coverage for the tests and updated/added some tests to improved the coverage.

### CODE REVIEW

**Comment by bjornhellander** [comment]
Do you have time to review, @sharwell? Only test improvements.

**Comment by bjornhellander** [comment]
Do you have time to review, @sharwell?

**Comment by sharwell** [comment]
@bjornhellander I'm about to submit a large change to the tests and would prefer to review this after it is merged. There may be some conflicts that need to be resolved in this PR, but it should be fairly minor compared to altering my other change.

**Comment by bjornhellander** [comment]
Ok. No problem. I'll merge and push again when you are done. Just let me know when.

File: StyleCop.Analyzers/StyleCop.Analyzers.Test/NamingRules/SA1304UnitTests.cs
Comment (sharwell) [inline StyleCop.Analyzers/StyleCop.Analyzers.Test/NamingRules/SA1304UnitTests.cs:65]:
```suggestion
        public async Task TestFieldInNativeMethodsClassAsync()
```

File: StyleCop.Analyzers/StyleCop.Analyzers.Test/NamingRules/SA1305UnitTests.cs
Comment (sharwell) [inline StyleCop.Analyzers/StyleCop.Analyzers.Test/NamingRules/SA1305UnitTests.cs:66]:
```suggestion
        public async Task TestVariableNamesInNativeMethodsClassAsync()
```

**Comment by sharwell** [comment]
Looks good once the conflict is resolved.

**Review by sharwell — APPROVED** [review]

**Pull request merged**
